docs/developers/manual.md

```diff
@@ -26,12 +21,12 @@ DATABASE_URL="sqlite3:/$HOME/.sematic/db.sqlite3"
 Then create the database:
 
 ```shell
-$ dbmate create
+$ touch $DATABASE_URL
```

Comment:
`DATABASE_URL` holds something like `sqlite3:/$HOME/.sematic/db.sqlite3`. You need something with the `sqlite3` prefix stripped